lib/core/themes/app_images.dart

```diff
@@ -1,4 +1,8 @@
 class AppImages {
```

Comment:
why AppImages class in themes folder ?

I guess it's better in constants folder or something like this.